addons/test_discuss_full/tests/test_performance.py

```diff
@@ -1052,3 +1052,24 @@ def _get_query_count(self):
             The point of having a separate getter is to allow it to be overriden.
         """
         return 75
+
+    def test_activities_for_lead_lost_opportunity(self):
+        """
+            Checks if the activities made on the lost lead is shown in the
+            systray activities.
+        """
+        lead = self.env['crm.lead'].create({'name': 'TEST LEAD'})
```

Comment:
Don't use real models in tests. You can use test models available in test modules; moreover the purpose of this test is to check the active flag management, so simply pick a model with active flag, set a record as inactive and check the result, no need to do the whole lead flow :) .